src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
add target

Reply:
isn`t target adding optional?

Reply:
it's optional in the sense that the annotation will work without it, but it's recommended to specify it to restrict and document the usage of the annotation correctly